src/app/game-state/activity.service.ts

```diff
@@ -542,11 +542,22 @@ export class ActivityService {
       consequence: [() => {
         this.characterService.characterState.status.stamina.value -= 100;
         let oreValue = 0;
+        let builderPower = 0;
         for (let i = 0; i < 200; i++){
           oreValue = this.inventoryService.consume('ore');
         }
+        for (let follower of this.followerService.followers){
+          if (follower.job = "builder"){
```

Comment:
You need to be careful  with == vs =. This loop would turn all your followers into builders every time it ran.

Reply:
Ooh, that's bad, because I think I copy pasted that from something, I'll doublecheck where I got it.

Reply:
Yeah, check line 631 of the base: https://github.com/Hornsoup/immortalityidle.github.io/blob/26b2602eea164931b5cffbdcd54737ff1df789f8/src/app/game-state/activity.service.ts#L619

Reply:
Hahahaha! What a fun way to catch a bug.